src/test/java/test/HelmOutputComparisonTest.java

```diff
@@ -10,78 +14,43 @@
 
 class HelmOutputComparisonTest {
```

Comment:
This is the original unit test that just compared the entire Helm YAML output with an expectation file. It's useful as a baseline, so I'm keeping it for now. I've just refactored it to use the extracted `Helm` utlity class.